src/kpro_auth_backend.erl

```diff
@@ -0,0 +1,38 @@
+%%%
+%%%   Copyright (c) 2017-2018, Klarna AB
```

Comment:
As a new file, shouldn't the copyright just be 2018?

Reply:
this file and kpro_connection.erl are copied from `brod.git`